content/en/community/contributing/creating-good-first-issues.md

```diff
@@ -20,6 +20,25 @@ Follow the next steps to create a "Good First Issue":
   These are not the same as `Help wanted` tickets - be sure you [know the difference](/community/contributing/first-time-contributors/#your-first-issue).
 {{< /callout >}}
 
+## Markdown template
+
+Use this content to paste into a GitHub ticket.  The add the steps needed.  See [the ticket below](#example-ticket) for an example of what this would look like:
```

Comment:
```suggestion
Use this content to paste into a GitHub ticket.  The add the steps needed to solve the ticket.  See [the ticket below](#example-ticket) for an example of what this would look like:
```